src/api/app/controllers/webui/kiwi/images_controller.rb

```diff
@@ -68,6 +62,10 @@ def set_image
         flash[:error] = "KIWI image '#{params[:id]}' does not exist"
         redirect_back(fallback_location: root_path)
       end
+
+      def set_repositories
+        @repositories = @image.repositories.order(:order)
```

Comment:
but the order is really important, so we want to ensure that it is done, it doesn't hurts. :wink: